### PR TITLE
Update ubuntu VMImage to ubuntu-20.04

### PR DESCRIPTION
**Pull request by chidozieononiwu**
Should resolve https://github.com/Azure/azure-sdk-tools/issues/1321

### CODE REVIEW

File: .azure-pipelines/warden.yml [hunk 1]
```diff
@@ -72,7 +72,7 @@ jobs:
   dependsOn: 'Build'
 
   pool:
-    vmImage: 'ubuntu-16.04'  
+    vmImage: 'ubuntu-20.04'
```
Comment (weshaggard) [inline .azure-pipelines/warden.yml:75]: Did you choose 20.04 over 18.04 for a particular reason?

Comment (weshaggard) [reply .azure-pipelines/warden.yml:75]: We should consider sticking to 18.04 until we attempt to move everything to 20.04. However if this is the start of that attempt I'm fine with what you have.

**Comment by weshaggard** [comment]
This only resolves a small part of #1321. The larger work item is to move everything across all our repos.

File: .azure-pipelines/warden.yml (hunk 1 above)
Comment (mitchdenny) [reply .azure-pipelines/warden.yml:75]: I think it might have been because of the comment @kurtzeborn made on the linked issue.

Comment (chidozieononiwu) [reply .azure-pipelines/warden.yml:75]: @mitchdenny Yes it due to Scotts comment. Should I go ahead with 20.0.4?

Comment (mitchdenny) [reply .azure-pipelines/warden.yml:75]: I'd probably go with 18.04 as its our primary Linux image right now unless there was an explicit reason for going 20.04.

**Pull request merged**